### ISSUE TITLE
Test Coverage Improvement

### ISSUE THREAD

**Issue by ElderJames** [ant-design-blazor/ant-design-blazor#2644]
We plan to release the official 1.0 version when the coverage reaches the desired value > 90%.

The following are all of the components currently in ant-design. Let's add test for them together.

- [ ] affix  
- [ ] alert 
- [ ] anchor 
- [ ] auto-complete  
- [ ] avatar  
- [ ] back-top  
- [ ] badge  
- [ ] breadcrumb
- [ ] button 
- [ ] calendar  
- [ ] card	 
- [ ] carousel 
- [ ] cascader 
- [ ] checkbox 
- [ ] collapse 
- [ ] comment  
- [ ] date-picker  
- [ ] descriptions  
- [ ] divider  
- [ ] drawer 
- [ ] dropdown 
- [ ] empty 
- [ ] form 
- [ ] grid 
- [ ] icon  
- [ ] input-number 
- [ ] input 
- [ ] layout 
- [ ] list  
- [ ] mentions  
- [ ] menu 
- [ ] message 
- [ ] modal  
- [ ] notification 
- [ ] page-header  
- [ ] pagination 
- [ ] popconfirm  
- [ ] popover 
- [ ] progress  
- [ ] radio 
- [ ] rate  
- [ ] result 
- [ ] segmented
- [ ] select 
- [ ] skeleton  
- [ ] slider 
- [ ] space 
- [ ] spin 
- [ ] statistic 
- [ ] steps 
- [ ] switch 
- [ ] table 
- [ ] tabs 
- [ ] tag 
- [ ] time-picker 
- [ ] timeline 
- [ ] tooltip 
- [ ] transfer 
- [ ] tree-select 
- [ ] tree 
- [ ] typography 
- [ ] upload

**Comment by kooliokey** [comment]
@ElderJames That PR was for Alert that just needs moved in the list from affix

**Comment by ElderJames** [comment]
@kooliokey Fixed. Thank you.

**Comment by ElderJames** [comment]
Hello @kooliokey , would you like to continue to help the improvement?

**Comment by ElderJames** [comment]
Hello @kooliokey , we still need to improve coverage, we'd appreciate it if you were available to help! 😊

**Comment by ElderJames** [comment]
We also can use Playwright to setup E2E tests for the complex components, do you have any experience with this?  @kooliokey

**Comment by kooliokey** [comment]
> We also can use Playwright to setup E2E tests for the complex components, do you have any experience with this?  @kooliokey 

That sounds like a good idea. I haven't used playwright before but have wanted to try it. Would we be testing the docs site with it? You have to run an application not random code from what I remember.

**Comment by ElderJames** [comment]
Yes, we need to add a standalone E2E project for testing, because the demos on the documentation site don't cover the full use cases.
